src/dom_util.js

```diff
@@ -68,7 +68,7 @@ const getRoot = function(node) {
  *     corresponding to node, if present.
  */
 const getActiveElement = function(node) {
-  const root = getRoot(node);
+  const root = getRootNode.call(node);
```

Comment:
I'm pretty sure that's because of the `o.f.call(o`, which can be optimized. The `getRootNode.call(node);` [can't](https://jsperf.com/getrootnode-vs-call).